src/components/Navbar/MenuItems.js

```diff
@@ -0,0 +1,27 @@
+export const MenuItems = [
+    {
+        title: 'Home',
+        url: '#',
+        cName: 'nav-links'
+    },
+    {
+        title: 'Blog Posts',
+        url: '#',
+        cName: 'nav-links'
+    },
+    {
+        title: 'Pictures',
+        url: '#',
```

Comment:
you could change these and link to important places! like your github or your email for the contact